app/_layouts/landing_page.html

```diff
@@ -2,17 +2,29 @@
 layout: default
 ---
 
-<main class="flex flex-col gap-14 pt-12">
-  {% for rows in page.config %}
-  <div class="grid grid-cols-1 lg:grid-cols-{{ rows.row | size }} gap-2">
-    {% for column in rows.row %}
-    <div class="flex flex-col">
-      {% for entry in column.column %}
-      {% assign include_path = "landing_pages/" | append: entry.type | append : ".md" %}
-      {% include {{ include_path }} type=entry.type config=entry.config %}
+<main class="w-full">
+  {% for row in page.rows %}
+    <div class="lp-row mb-8">
```

Comment:
@mheap what's `lp-col` and `lp-row`?

Reply:
Landing Page COLumn and Landing Page ROW - they were markers to help me identify divs in the output whilst I was working. There's no need for them to be there now. Feel free to remove